azerothcore/azerothcore-wotlk
# fix(Core/Spells): Hand of/Divine Sacrifice should remove remove the Hungering Cold frozen effect.

## Changes Proposed:
-  Hand of/Divine Sacrifice now remove the Hungering Cold frozen effect.


## Issues Addressed:
- Closes 



## Tests Performed:

- Tested on windows 10 and ubuntu 20.


## How to Test the Changes:

- Log in with a paladin and any other character.
- Make a group with the two characters.
- While any npc is hitting his other character cast with the paladin `` Hand of sacrifice`` or `` Divine sacrifice``.
- Apply `` Hungering cold`` on yourself and verify that the damage generated by ``Divine sacrifice`` or ``Hand of sacrifice`` eliminates the effect of `` Hungering Cold``


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
